src/transformers/__init__.py

```diff
@@ -1455,6 +1457,18 @@
             "ErniePreTrainedModel",
         ]
     )
+    _import_structure["models.ernie_m"].extend(
+        [
+            "ERNIE_M_PRETRAINED_MODEL_ARCHIVE_LIST",
+            "ErnieMForMultipleChoice",
+            "ErnieMForQuestionAnswering",
+            "ErnieMForSequenceClassification",
+            "ErnieMForTokenClassification",
+            "ErnieMModel",
+            "ErnieMPreTrainedModel",
+            "ErnieMUIEM",
```

Comment:
If this ends up being public, it needs a better name. What does UIEM stand for?

Reply:
UIEM stands for `Universal Information Extraction Model` (it was implemented in the original paddlenlp implementation of Ernie-M) [here](https://github.com/PaddlePaddle/PaddleNLP/blob/7e843eb1bb9e838ba80689ac80575f90e8835f5a/paddlenlp/transformers/ernie_m/modeling.py#L774)

Reply:
Maybe `ErnieMForInformationExtraction` then. It will be more understandable to a user than UIEM